content/news/playtest-20150808.md

```diff
@@ -0,0 +1,58 @@
+
+Other changes include:
+
+* Spies in the Red Alert mod have received a major overhaul, making them much more useful.
+* Paratroopers can now be selected and given orders before they land.
+* Two new single-player missions have been added, Soviets 04a and 04b.
+* Single-player missions have been polished, and Allies 05 has received a new ultra-hard difficulty.
+* Mission replays can again be viewed through the replay browser.
+* A warning message is now given when a server admin selects a map with custom rules.
+* Range circles are now visible to allied players and spectators.
+* Players can now hide the in-game UI (using a hotkey, bound to Ctrl+Shift+H by default).
+* We now support installing game content from *The First Decade* DVD.
+
+Of interest for modders and mappers will be the addition of new Lua API functions.  Disguising spies, capturing or infiltrating units and buildings using scripting is now possible.  Other newly available functions can be used to play arbitrary sound files, flash the screen, and control the new palette effect trait.
+
+Of course we did not only add new things, but also fixed bugs and annoyances.  Here's a small sample:
+
+* A number of crashes have been resolved.
+* The score screen can now be closed to have a look at the battlefield after the match.
+* The 'Options' button will no longer blink without reason in skirmish maps.
+* The duration bar on chronoshifted units in the Red Alert mod is no longer visible to enemy players.
+* The aircraft and helicopter cruise altitude in the Red Alert mod has been increased again.
+* The unit duplication crate bonus has been removed from the Tiberian Dawn mod.
+
+For all the gritty details check out the [full changelog](http://changelog.openra.net).
+
+Grab the installer for this playtest [from our download page](/download/) and give it a try!
+
+<div style="text-align:center" markdown="1">
+
+![Custom map rule warning](/images/news/20150808-cnc-lobby.png)
+
+The game will warn players in the lobby when the host selects a map with custom rules.  This should help combat the small number of dishonest players, who adjusted rules to their advantage.
+
+![Veteran unit production after infiltration](/images/news/20150808-ra-infiltrate-production-bonus.png)
+
+Infiltrating an enemy production building with a spy will let you build veteran units.
+
+![Observer range circles and hidden gui](/images/news/20150808-d2k-range-circles.png)
+
+Allied players and spectators can now see the range circles from units and buildings.
+
+<iframe width="600" height="333" src="https://www.youtube-nocookie.com/embed/3ai-MeYDx5c?rel=0" frameborder="0" allowfullscreen></iframe>
+
+New Lua API functions can be used to simulate simple weather effects.
+
+<iframe width="600" height="333" src="https://www.youtube-nocookie.com/embed/WoZoQILTaGk?rel=0" frameborder="0" allowfullscreen></iframe>
+
+Work continues on our Tiberian Sun mod.  Check out these clips from a recent test game!
```

Comment:
For the sake of our inboxes I think we should add a disclaimer that the TS mod still isn't quite ready to be shipped by default.